src/styles/index.scss

```
@@ -1,3 +1,63 @@
+$column-width: 100px;
+$height: 100px;
+$gap: 1px;
+$padding: 10px;
+$week-days: 0 'mon', 1 'tue', 2 'wed', 3 'thu', 4 'fri', 5 'sat', 6 'sun';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar{
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  gap: 1px;
+  width: calc(7 * #{$column-width} + 2 * #{$padding});
+}
+
+.calendar__day{
+  width: calc((100% - 6 * 1px - 2 * #{$padding}) / 7);
+  height: $column-width;
+  background-color: #eee;
+  border: 1px solid black;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition: background-color 0.5s, transform 0.5s;
+}
+
+.calendar__day:hover{
+  cursor: pointer;
+  background-color: #FFBFCB;
+  transform: translateY(-20px);
+}
+
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::before {
+    content: '#{$i}';
+  }
+}
```

you can use SCSS nesting here as well

put it inside `.calendar {}` brackets and:
```suggestion
@for $i from 1 through 31 {
  &__day:nth-child(#{$i})::before {
    content: '#{$i}';
  }
}
```